src/IOUtils.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
```

Comment:
lets move this file to either `test/` directory, or a new directory for local rendering